docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+#
+# docker-compose.yml
+#
+# A simple deployment of Synse Server with an emulator plugin
+# instance. Useful for learning, testing, and debugging.
+#
+
+version: '3.4'
```

Comment:
yeah, we need 3.4 (https://docs.docker.com/compose/compose-file/compose-versioning/#version-34) since the composefile  uses the `start_period` option for health checks